### DISCUSSION TITLE
Only show tab number when keyboard modifier is held...

### DISCUSSION THREAD

**Discussion by selvinkuik** [kg8m/chrome-show-tab-numbers#104] [Ideas]
I think this extension is great! However, my primary use for it is to quickly switch between tabs. For example, on Mac by holding down COMMAND and then the tab number. On Windows I would assume the same functionality is provided by holding CONTROL. Based on this, would it be possible to add an option that only shows the tab number when COMMAND (or CONTROL on Windows) is held down?

**Comment by kg8m** [comment]
@selvinkuik 
Thank you for your feedback. I think it is difficult to enable tab numbering while holding down a key, but you can configure a keyboard shortcut to toggle tab numbering. I hope you’ll consider trying the feature if you’re interested.

**Comment by selvinkuik** [comment]
Oh right, thanks for letting me know. Toggling would be fine, but I think I'm being really thick. I think I see the option to toggle tab numbering, but how do I configure this to work via a keyboard shortcut?

![Screenshot 2024-09-21 13 38 50](https://github.com/user-attachments/assets/9fe15201-d7ad-4191-af8a-78d86f150be9)

**Reply by kg8m** [reply]
I am presenting a web page that explains how to customize keyboard shortcuts for Chrome extensions. You can find similar explanations on Google, ChatGPT, and other platforms.

[Customize keyboard shortcuts for browser extensions](https://copytonotion.com/guides/customize-keyboard-shortcuts-for-browser-extensions)

> Chromium-based browsers like **Google Chrome, Microsoft Edge, and Brave** allow users to customize extensions' keyboard shortcuts for a more personalized browsing experience. Here's how to do it:
> 
> 1. Access Extension Settings: Open your browser's extension settings.
>     - In Chrome, enter **chrome://extensions/shortcuts** in the address bar.
>     - For Edge, go to **edge://extensions/shortcuts**
>     - For Brave, visit **brave://extensions/shortcuts**
> 2. Locate the extension: Scroll through the list of installed extensions to find the extension.
> 3. Edit Shortcuts: Click on the edit pencil button next to the extension.
>     ![Screenshot_20230722_130601](https://github.com/user-attachments/assets/3d874f7f-5c68-4ad0-84e0-fba2841f7c11)
> 4. Enter your desired shortcut or leave it blank to disable the existing shortcut.

**Reply by selvinkuik** [reply]
Thanks @kg8m – that's awesome!
